### DISCUSSION TITLE
How to clear out password on windows command line to use PAT?

### DISCUSSION THREAD

**Discussion by MPV65** [community/community#21614] [New to GitHub]
<p>Many of us will have had the notification from GitHub that passwords are being deprecated in favour of personal access tokens. However, to use a PAT on the command line, one must first clear one’s existing password so that git prompts for it. Whilst there is help on how to do this on MacOS on the GitHub pages, there is no help I can find about how to do this in Windows.</p>
<p>Could anyone give a clear explanation about how to do this?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>I have requested specific documentation so I’ll add your ask to the list as well :slight_smile:</p>
<p>First you need to know where you’ve stored your credentials.</p>
<p>Run:</p>
<pre><code> git config --list
</code></pre>
<p>One of the entries should be <code>credential.helper=</code> if that is <code>manager</code> then you are using the Windows Credential Manager.</p>
<p>You can directly edit the Credential Manager entry for GitHub by finding Credential Manager in your Control Panel.</p>
<p>If Credential Manager wasn’t the answer let us know what the output of <code>credential.helper</code> was.</p>

**Comment by MPV65** [comment]
<p>Thanks. I have now updated my details in Windows Credential Manager.</p>
